.github/workflows/buildPages.yml

```diff
@@ -7,8 +7,6 @@ on:
   # Triggers the workflow on push or pull request events but only for the "main" branch
   push:
     branches: [ "main" ]
-  pull_request:
```

Comment:
This change was part of the typedoc building task that @DanielBelmes implemented for the website. Please don't commit to `main` until he reviews it, and confirms that this won't break his work on that part of the CI. https://github.com/EtherealEngine/etherealengine-docs/commit/fade44530d77b7e8d4a2caed5d15665fee576d53

Reply:
Yes sure
@DanielBelmes I removed it because I thought anybody will be able to change our docs by creating a pull request which is not what we want to happen

Reply:
There's if conditions to disable the deploy if it's a PR. So it just checks if you can build. It should be a separate check action but I was playing around with actions 😅

Reply:
Sorry, missed that line. Reverting this line.

Reply:
Resolved by https://github.com/EtherealEngine/etherealengine-docs/pull/38/commits/7850f66e600d6e41d3fe3a6aabae920ffe183961